CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/DelimitedImportController.java

```diff
@@ -215,6 +220,65 @@ protected void updateSampleData() {
         }
     }
 
+    private void notifyFileStructureMismatchWarning(final List<File> filesToRemove) {
+        final StringBuilder warningMsg = new StringBuilder((filesToRemove.size() > 1 ? "These files have" : "This file has")
+                + " a different structure and will be deleted: " + SeparatorConstants.NEWLINE);
+
+        filesToRemove.forEach(file -> warningMsg.append(file.getPath() + SeparatorConstants.NEWLINE));
+
+        LOGGER.log(Level.INFO, "Header structure mismatch. These files will be removed: {0}", filesToRemove);
+
+        NotifyDisplayer.displayAlert("Header structure mismatch", "The header structure of the files should be the same", "If the `Files Include Headers` "
+                + "option is disabled the number of columns should be the same. Otherwise the column headers should be the same in the same order. "
```

Comment:
Added some commas and very minor updates to the sentence for your thoughts;

`"option is disabled, the number of columns should be the same. Otherwise the column headers should be the same, in the same order. "`